src/CopyrighFixer/CopyrighFixer/bar.h

```diff
@@ -0,0 +1,19 @@
+#ifndef BAR_H
+#define BAR_H
+
+#include <QString>
+#include <iostream>
+
```

Comment:
You forget add doxygen documentation for your class.
and all them methods.

Reply:
Done